Microsoft.Toolkit.Forms.UI.XamlHost/Microsoft.Toolkit.Forms.UI.XamlHost.csproj

```diff
@@ -1,10 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFrameworks>net462;netcoreapp3.0</TargetFrameworks>
```

Comment:
Why net462? XamlHosts require RS5+, so I guess we can target net472. When we get into 19H1 net48 will be the default.

Reply:
We don't want to force a dev to upgrade the app's .net framework version just because this specific feature (xaml islands) will always run on a pc that has 4.7.2 installed. We should support the min version that supports the feature. If we just update to net 472, any developer that is targeting net 462 and Win7 (for example), would be unable to use this feature, on a scenario where the dev detects if the app is running on Win7 or Win10 and gives a different UI based on that.

Reply:
I'd love to see how to use XAML adaptive coding to enable the Win7 scenario. I remember this [hack](https://github.com/qmatteoq/DesktopBridgeHelpers/blob/master/DesktopBridge.Helpers/Helpers.cs) that also required carefully coding to avoid JIT issues.

In the other hand, I think that requiring 4.7.2 is a good-enough check to avoid the app to run on older Win10 without XamlIslands.